### ISSUE TITLE
How to deal different languages

### ISSUE THREAD

**Issue by naticaceres** [railsstudent/ng-spanish-menu#2]
Are we insterested in using i18n? 
transloco? 
ngx-translate?
since plenty of names are already in english, and some of them are in spanish, wouldn't it be cool if we could make it available for both?

**Comment by railsstudent** [comment]
Yes.   Use transloco because the maintainer of ngx-translate does not maintain the project anymore.

**Comment by railsstudent** [comment]
We can add a language dropdown where people can select a language and read localized texts.
